elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -80,26 +80,30 @@ interface FilterPeersOptions {
 	readonly asc: boolean;
 }
 
-const filterPeersByCategory = (
+// Returns an array of peers to be protected
+export const filterPeersByCategory = (
```

Comment:
Can we make this function to accept argument `sort`, that takes `ASC` and `DESC` enum instead of because when we give `asc` as `false` then it should not mean that we have to return descending. Also, in the framework api module has a sort field that accepts `ASC` and `DESC`.

Reply:
Maybe instead of `asc` or `desc` it can just be an enum `highest` or `lowest` because it's not clear from outside how the sorting relates to the selection (at first, it's not clear if the function protects the first `x%` or the last `x%`? If the developer didn't read the implementation of the function, they won't know whether they should use asc or desc so `highest` and `lowest` is more clear).